shared/AppInsightsCore/Tests/Selenium/CorePerfCheck.Tests.ts

```diff
@@ -2,7 +2,7 @@ import { Assert, AITestClass } from "@microsoft/ai-test-framework";
 import { ITestContext, StepResult } from "@microsoft/ai-test-framework/dist-esm/src/TestCase";
 import { _InternalMessageId, LoggingSeverity } from "../../src/JavaScriptSDK.Enums/LoggingEnums";
 import { _InternalLogMessage, DiagnosticLogger } from "../../src/JavaScriptSDK/DiagnosticLogger";
-import { isObject, objForEachKey, objForEachKey as objKeys, optimizeObject, setValue } from "../../src/JavaScriptSDK/HelperFuncs";
+import { isObject, objForEachKey, objKeys, optimizeObject, setValue } from "../../src/JavaScriptSDK/HelperFuncs";
```

Comment:
Note quite sure when this change happened, but this effective made the objKeys() tests objForEachKey() -- duplicating the tests...